terraform/aws/modules/module_ecs_service/sg_rules_dlt.tf

```diff
@@ -0,0 +1,33 @@
+# vpc peering to the network where DLT is hosted
+
+data "aws_vpc" "dlt_vpc" {
+  id = var.dlt_vpc_id
+}
+
+# DLT traffic - From the Distributed load tester to the load balancer
+
+resource "aws_security_group_rule" "dlt_ingress_rule_to_loadbalancer" {
```

Comment:
Not exactly the way I intended it to be done but if it works it can stay, but:
1, Make it optional with count parameter, not all environments require DLT connection
2. The ports should not be hardcoded, the port should be the same as the listener port in LB, use the same variable as in LB listener

Reply:
What do you think?

Reply:
The port is ok
The count needs to be added

Reply:
Done